dll/win32/getuname/lang/ro-RO.rc

```diff
@@ -3662,262 +3662,262 @@ BEGIN
-    4538 "Hangul jongseong sios"
-    4539 "Hangul jongseong ssangsios"
-    4540 "Hangul jongseong ieung"
-    4541 "Hangul jongseong cieuc"
-    4542 "Hangul jongseong chieuch"
-    4543 "Hangul jongseong khieukh"
-    4544 "Hangul jongseong thieuth"
-    4545 "Hangul jongseong phieuph"
-    4546 "Hangul jongseong hieuh"
-    4547 "Hangul jongseong kiyeok-rieul"
-    4548 "Hangul jongseong kiyeok-sios-kiyeok"
-    4549 "Hangul jongseong nieun-kiyeok"
-    4550 "Hangul jongseong nieun-tikeut"
-    4551 "Hangul jongseong nieun-sios"
-    4552 "Hangul jongseong nieun-pansios"
-    4553 "Hangul jongseong nieun-thieuth"
-    4554 "Hangul jongseong tikeut-kiyeok"
-    4555 "Hangul jongseong tikeut-rieul"
-    4556 "Hangul jongseong rieul-kiyeok-sios"
-    4557 "Hangul jongseong rieul-nieun"
-    4558 "Hangul jongseong rieul-tikeut"
-    4559 "Hangul jongseong rieul-tikeut-hieuh"
-    4560 "Hangul jongseong ssangrieul"
-    4561 "Hangul jongseong rieul-mieum-kiyeok"
-    4562 "Hangul jongseong rieul-mieum-sios"
-    4563 "Hangul jongseong rieul-pieup-sios"
-    4564 "Hangul jongseong rieul-pieup-hieuh"
-    4565 "Hangul jongseong rieul-kapyeounpieup"
-    4566 "Hangul jongseong rieul-ssangsios"
-    4567 "Hangul jongseong rieul-pansios"
-    4568 "Hangul jongseong rieul-khieukh"
-    4569 "Hangul jongseong rieul-yeorinhieuh"
-    4570 "Hangul jongseong mieum-kiyeok"
-    4571 "Hangul jongseong mieum-rieul"
-    4572 "Hangul jongseong mieum-pieup"
-    4573 "Hangul jongseong mieum-sios"
-    4574 "Hangul jongseong mieum-ssangsios"
-    4575 "Hangul jongseong mieum-pansios"
-    4576 "Hangul jongseong mieum-chieuch"
-    4577 "Hangul jongseong mieum-hieuh"
-    4578 "Hangul jongseong kapyeounmieum"
-    4579 "Hangul jongseong pieup-rieul"
-    4580 "Hangul jongseong pieup-phieuph"
-    4581 "Hangul jongseong pieup-hieuh"
-    4582 "Hangul jongseong kapyeounpieup"
-    4583 "Hangul jongseong sios-kiyeok"
-    4584 "Hangul jongseong sios-tikeut"
-    4585 "Hangul jongseong sios-rieul"
-    4586 "Hangul jongseong sios-pieup"
-    4587 "Hangul jongseong pansios"
-    4588 "Hangul jongseong ieung-kiyeok"
-    4589 "Hangul jongseong ieung-ssangkiyeok"
-    4590 "Hangul jongseong ssangieung"
-    4591 "Hangul jongseong ieung-khieukh"
-    4592 "Hangul jongseong yesieung"
-    4593 "Hangul jongseong yesieung-sios"
-    4594 "Hangul jongseong yesieung-pansios"
-    4595 "Hangul jongseong phieuph-pieup"
-    4596 "Hangul jongseong kapyeounphieuph"
-    4597 "Hangul jongseong hieuh-nieun"
-    4598 "Hangul jongseong hieuh-rieul"
-    4599 "Hangul jongseong hieuh-mieum"
-    4600 "Hangul jongseong hieuh-pieup"
-    4601 "Hangul jongseong yeorinhieuh"
-    4602 "Hangul jongseong kiyeok-nieun"
-    4603 "Hangul jongseong kiyeok-pieup"
-    4604 "Hangul jongseong kiyeok-chieuch"
-    4605 "Hangul jongseong kiyeok-khieukh"
-    4606 "Hangul jongseong kiyeok-hieuh"
-    4607 "Hangul jongseong ssangnieun"
+    4352 "Hangulul choseong kiyeok"
```

Comment:
Since Hangul are letters, won't be better to have it "Litera ..." (like how are for others in the lines above)?